### PR TITLE
created EnergyCurve Display plugin

### PR DESCRIPTION
**Pull request by modukon**
Shows the energy curve of the selected planet.
You can hover over the curve to set a point and find out
when exactly this point will be reached.

Can be used to find out when a foundry reaches 95%.
Helps to understand how energy growth works.

### CODE REVIEW

**Comment by Robiquet** [comment]
Really cool! Would be good if the energy growth was quantified and displayed too. I've been wondering if upgrading a planet is good for energy growth in the short term as it increases the energy cap, which could push it down to the 'slower' end of the s curve

**Review by Stx69 — DISMISSED** [review]
uuuuu 25% of sine wave... NICE!

File: content/utilities/energy-curve-display/plugin.js [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+	
+	let predictedEnergy = null;
+	let predictedEnergyPercent = null;
+	let predictedTimeSec = null;
+	let predictedTime = null;
+	
+	let intervUpdate = null;
+	
+	o.init = function() {
+		let planets = df.getMyPlanets();
```
Comment (jacobrosenthal) [inline content/utilities/energy-curve-display/plugin.js:218]: This isnt doing anything?
```suggestion
```

**Review by jacobrosenthal — APPROVED** [review]
I have a feeling its a bit bigger than it needs to be with current featureset, but can be dialed in later with user feedback.

Really nice education tool

**Pull request merged**